Automattic/wp-calypso
# Launch: Move launch store to package/data-stores.

#### Changes proposed in this Pull Request

* Move launch store to `package/data-stores`.
* Also fixed several type issues with launch stores.

#### Testing instructions

* Run `yarn start`.
* Run `yarn dev --sync` on `apps/editing-toolkit`.
* Test on sandbox.
* Launch flow should work as usual.
  * Also test for data persistence, values like selected domain, searched domain keyword, and selected plan should be remembered.
* Additionally, test gutenboarding `/new` for any possible regression (unlikely but hey).
